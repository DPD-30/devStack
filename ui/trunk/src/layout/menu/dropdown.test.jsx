import React from 'react'
import { renderToString } from 'react-dom/server'

import LayoutMenuDropdown from './dropdown.jsx'

jest.unmock('./dropdown.jsx')


describe('', () => {
    it('LayoutMenuDropdown should match snapshot', () => {
        const comp = renderToString(
            <LayoutMenuDropdown
                label="Mock label"
            >
                mock children
            </LayoutMenuDropdown>
        )

        expect(comp).toMatchSnapshot()
    })
})
