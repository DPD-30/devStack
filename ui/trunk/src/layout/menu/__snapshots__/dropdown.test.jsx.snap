// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LayoutMenuDropdown should match snapshot 1`] = `
<li class="dropdown nav-item">
  <a aria-haspopup="true"
     href="#"
     class="dropdown-toggle nav-link"
     aria-expanded="false"
  >
    Mock label
  </a>
  <div tabindex="-1"
       role="menu"
       aria-hidden="true"
       class="dropdown-menu"
  >
    mock children
  </div>
</li>
`;
